### PR TITLE
stashed intermediate ca certs get more identifiable names

### PR DESCRIPTION
**Pull request by lziest**
- Add SKI or serial to help us identify different intermediate CA certs
  with same Common Name.

### CODE REVIEW

**Comment by kisom** [comment]
LGTM.

**Comment by jacobhaven** [comment]
Do we need the `fileName += fmt.Sprintf(".%d", time.Now().UnixNano())` at line 379?

File: bundler/bundler.go [hunk 1]
```diff
@@ -426,6 +426,15 @@ func (b *Bundler) fetchIntermediates(certs []*x509.Certificate) (err error) {
 			// construct the filename as the CN with no period and space
 			name = strings.Replace(cert.Subject.CommonName, ".", "", -1)
 			name = strings.Replace(name, " ", "", -1)
+
+			// add SKI as extra identifier
+			// in case SKI is nil, use serial number as extra identifier
+			if cert.SubjectKeyId != nil {
+				name += fmt.Sprintf("_SKI_%x", cert.SubjectKeyId)
+			} else {
+				name += fmt.Sprintf("_serial_%x", cert.SerialNumber.Bytes())
```
Comment (jacobhaven) [inline bundler/bundler.go]: It might make sense to just ignore the serial number, and append a (truncated) hash of the `cert.Raw`.

Comment (kisom) [inline bundler/bundler.go]: I don't know that it's compelling enough to change, though.

**Comment by lziest** [comment]
Per OOB discussion, use SKI _and_ serial number as extra identifier, stop using timestamp.

**Comment by jacobhaven** [comment]
LGTM

**Pull request merged**